x-pack/test_serverless/functional/test_suites/search/search_index_detail.ts

```diff
@@ -24,210 +24,252 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
+            await pageObjects.svlSearchIndexDetailPage.expectIndexDetailPageHeader();
+            await pageObjects.svlSearchIndexDetailPage.expectUrlShouldChangeTo('settings');
+          });
+          it('navigates to mappings tab', async () => {
+            await pageObjects.indexManagement.changeManageIndexTab('showMappingsIndexMenuButton');
+            await pageObjects.svlSearchIndexDetailPage.expectIndexDetailPageHeader();
+            await pageObjects.svlSearchIndexDetailPage.expectUrlShouldChangeTo('mappings');
+          });
+        });
+        describe('can view search index details', function () {
+          it('renders search index details with no documents', async () => {
+            await pageObjects.svlSearchIndexDetailPage.openIndicesDetailFromIndexManagementIndicesListTable(
+              0
+            );
+            await pageObjects.svlSearchIndexDetailPage.expectIndexDetailPageHeader();
+            await pageObjects.svlSearchIndexDetailPage.expectSearchIndexDetailsTabsExists();
+            await pageObjects.svlSearchIndexDetailPage.expectAPIReferenceDocLinkExists();
+          });
         });
       });
     });
-    describe('index management index details', () => {
+
+    describe('viewer', function () {
       before(async () => {
-        await es.indices.create({ index: indexName });
-        await security.testUser.setRoles(['index_management_user']);
-      });
-      beforeEach(async () => {
-        await pageObjects.common.navigateToApp('indexManagement');
-        // Navigate to the indices tab
-        await pageObjects.indexManagement.changeTabs('indicesTab');
-        await pageObjects.header.waitUntilLoadingHasFinished();
+        await esDeleteAllIndices(indexName);
+        await es.index({
+          index: indexName,
+          body: {
+            my_field: [1, 0, 1],
+          },
+        });
       });
       after(async () => {
         await esDeleteAllIndices(indexName);
       });
-      describe('manage index action', () => {
+      describe('search index details page', function () {
+        before(async () => {
+          await pageObjects.svlCommonPage.loginAsViewer();
+        });
         beforeEach(async () => {
-          await pageObjects.indexManagement.manageIndex(indexName);
-          await pageObjects.indexManagement.manageIndexContextMenuExists();
+          await svlSearchNavigation.navigateToIndexDetailPage(indexName);
         });
-        it('navigates to overview tab', async () => {
-          await pageObjects.indexManagement.changeManageIndexTab('showOverviewIndexMenuButton');
-          await pageObjects.svlSearchIndexDetailPage.expectIndexDetailPageHeader();
-          await pageObjects.svlSearchIndexDetailPage.expectUrlShouldChangeTo('data');
+        it('delete document button is disabled', async () => {
+          await pageObjects.svlSearchIndexDetailPage.expectDeleteDocumentActionIsDisabled();
```

Comment:
:x: We should also be testing that this is enabled in the developer tests.

Reply:
added in https://github.com/elastic/kibana/pull/197869/commits/21fbf903e93e029df109021ff874e83164c550c2